README.md

```diff
@@ -376,6 +378,28 @@ BigtableDataSettings.enableOpenCensusStats();
 BigtableDataSettings.enableGfeOpenCensusStats();
```

Comment:
I"m not sure that this line works still. Or at least it didn't compile when I tried using it I think

Reply:
Ah.. Looks like this change is not released yet